templates/default/services.conf.erb

```diff
@@ -170,17 +173,32 @@ filter {
 
 output {
   <% if @elastic_output -%>
-  opensearch {
-    hosts => [<%= @elastic_addr %>]
-    index => "<%= node['logstash']['service_index'] %>%{+YYYY.MM.dd}"
-    <% if node['elastic']['opensearch_security']['enabled'] %>
-    user => "<%=node['elastic']['opensearch_security']['logstash']['username']%>"
-    password => "<%=node['elastic']['opensearch_security']['logstash']['password']%>"
-    <% if node['elastic']['opensearch_security']['https']['enabled'] %>
-    cacert => "<%= @hops_ca %>"
-    ssl => true
-    <% end %>
-    <% end %>
+  if "onlinefs" in [tags] and [log_arguments][project_id] {
```

Comment:
I think it would be great if we can send them to both the project index and the main service log index so that admin can troubleshoot all the logs.